### DISCUSSION TITLE
Add slight delay when testing network for LXC

### DISCUSSION THREAD

**Discussion by DelusionalAI** [tteck/Proxmox#3478] [General] [answered]
Often when I try to create an LXC, I run into a slight issue where the CT gets a IPV6 IP address, says network connected and then a second later says no internet and asks me if I want to continue. Without fail if I wait 1 second and hit continue, it works, the script just seems to be testing for internet before the network is fully up. It is possible update the script so that if network is connected, but Internet fails, it waits for a second to test internet again automatically? 

I've attached a screenshot showing what I run into. 
![image](https://github.com/user-attachments/assets/4adcda44-06be-44d1-83d9-244a84e42f73)

**Comment by tteck** [comment]
Are you making use of IPv6?

**Reply by DelusionalAI** [reply]
Yes I am. IPv6 internet does also work on the containers and host, but like IPv4 it looks like it just isn't fully online when the test runs.

**Reply by tteck** [reply]
What exactly are you utilizing IPv6 for?

**Reply by DelusionalAI** [reply]
General networking? My entire network is IPv6 ready.

**Comment by tteck** [comment, marked as answer]
Check if it works for you now. https://github.com/tteck/Proxmox/commit/d26e474ccbf4bfcfc761ec82103321a3b44bb7c0

**Reply by DelusionalAI** [reply]
That did it, simple as that. 
![image](https://github.com/user-attachments/assets/da5f9e53-eb68-4d84-b8a3-1a54de81eca0)
